### ISSUE TITLE
about datase build

### ISSUE THREAD

**Issue by manik-123** [qiyunlab/HGTector#117]
when i run hgtector database -o db_dir --default
I get the below error
**************************************************
Done.
  Total number of genomes in categories: 288353.
Filtering genomes...
Traceback (most recent call last):
  File "/project/liuguohong/anaconda3/envs/hgtector/bin/hgtector", line 96, in <module>
    main()
  File "/project/liuguohong/anaconda3/envs/hgtector/bin/hgtector", line 35, in main
    module(args)
  File "/project/liuguohong/anaconda3/envs/hgtector/lib/python3.8/site-packages/hgtector/database.py", line 134, in __call__
    self.filter_genomes()
  File "/project/liuguohong/anaconda3/envs/hgtector/lib/python3.8/site-packages/hgtector/database.py", line 394, in filter_genomes
    self.df['accnov'] = self.df['accession'].str.split('.', 1).str[0]
  File "/project/liuguohong/anaconda3/envs/hgtector/lib/python3.8/site-packages/pandas/core/strings/accessor.py", line 129, in wrapper
    return func(self, *args, **kwargs)
TypeError: split() takes from 1 to 2 positional arguments but 3 were given

**Comment by manik-123** [comment]
I solved it

**Comment by manik-123** [comment]
> > I solved it
> 
> Hello，I‘m encountering the same problem. How do you solve it？ Looking forward your replay.
You may have to completely delete the conda environment and create it again. 
please use this......
conda create -n hgtector -c bioconda python=3.7.7 pyyaml=5.3.1 pandas=1.0.3 matplotlib=3.1.3 scikit-learn=0.22.1 diamond blast=2.9.0

**Comment by qiyunzhu** [comment]
Hello @manik-123 @zouminghong Thanks for reporting this bug. Sorry that I didn't get to it until now. Well I just fixed it (#118 ). It was a compatibility issue with newer versions of Pandas. You can update HGTector with:

```bash
pip install --force-reinstall --no-cache-dir git+https://github.com/qiyunlab/HGTector.git
```

Then the problem should be solved. Please let me know if that works for you. Thanks!

**Comment by manik-123** [comment]
Thank you so much

**Comment by qiyunzhu** [comment]
@manik-123 Just fixed another bug #119 You need to update again if you did so in the past few minutes :)
